pkg/daemon/drain.go

```diff
@@ -153,15 +152,17 @@ func (dn *Daemon) performDrain() error {
 	return nil
 }
 
+type ReadFileFunc func(string) ([]byte, error)
```

Comment:
This looks totally fine to me as is, but I'm thinking we may want to make this an interface in the future?

Reply:
Agreed. https://issues.redhat.com/browse/MCO-224